nuxt.config.js

```diff
@@ -184,6 +192,15 @@ export default {
 
   // Build Configuration: https://go.nuxtjs.dev/config-build
   build: {
+    extend(config) {
```

Comment:
do you get an error if this section is removed?

Reply:
@ermish Unfortunately, when I uncomment it the app doesn't build and I'm getting these errrors:
```
ERROR  Failed to compile with 6 errors                                                                                                                                           friendly-errors 21:56:17

These dependencies were not found:                                                                                                                                                friendly-errors 21:56:17
                                                                                                                                                                                  friendly-errors 21:56:17
* child_process in ./node_modules/google-auth-library/build/src/auth/googleauth.js                                                                                                friendly-errors 21:56:17
* fs in ./node_modules/google-auth-library/build/src/auth/googleauth.js, ./node_modules/google-p12-pem/build/src/index.js and 1 other                                             friendly-errors 21:56:17
* net in ./node_modules/https-proxy-agent/dist/agent.js                                                                                                                           friendly-errors 21:56:17
* tls in ./node_modules/https-proxy-agent/dist/agent.js                                                                                                                           friendly-errors 21:56:17
                                                                                                                                                                                  friendly-errors 21:56:17
To install them, you can run: npm install --save child_process fs net tls                                                                                                         friendly-errors 21:56:17
ℹ Waiting for file changes                                                                                                                                                                        21:56:17
ℹ Memory usage: 361 MB (RSS: 493 MB)                                                                                                                                                              21:56:17
ℹ Listening on: http://localhost:3000/
```
<img width="1167" alt="Screenshot 2021-07-18 at 21 56 35" src="https://user-images.githubusercontent.com/37947642/126067888-548e0e65-9592-456b-8884-273abeb9146a.png">